src/index.ts

```diff
@@ -19,3 +43,10 @@ export * from './EventPayload';
 export * from './EnumOrString';
 export * from './Action';
 export * from './VersionLabel';
+
+declare global {
```

Comment:
FYI: Without this global addition, I can not do `window['analyticsEventPayload'];` on line 20. It causes the error: 
`Element implicitly has an 'any' type because index expression is not of type 'number'.`

Reply:
Why do we need to save a payload to the `window` object in the first place?

Reply:
See other comment below.